photonpump/conversations.py

```diff
@@ -315,6 +315,12 @@ async def reply(self, message: InboundMessage, output: Queue) -> None:
         self.expect_only(message, TcpCommand.WriteEventsCompleted)
         result = proto.WriteEventsCompleted()
         result.ParseFromString(message.payload)
+        if result.result == proto.AccessDenied:
```

Comment:
this needs a sanity-check.  in the read methods, we delegate the logic inside the conversation's `reply()` to classes called, eg `ReadEventCompleted`, which then do a `.dispatch()`.   so if we wanted to make this more like the read events, we could create something like a `WriteEventCompleted` class.

 but it doesn't do much dispatching, compared to the read ones?  like it is either a ccess denied, or it does `self.set_result()`.

(i only have a very vague understanding of what's going on here)

Reply:
punted to #230